src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToArrayTransformer.php

```diff
@@ -72,14 +72,15 @@ public function transform($dateTime)
             ), array_flip($this->fields));
         }
 
-        if (!$dateTime instanceof \DateTime) {
-            throw new TransformationFailedException('Expected a \DateTime.');
+        // DateTimeInterface was introduced in PHP 5.5.0.
```

Comment:
@Tobion The comment explains why we have to check for both DateTime and DateTimeInterface. DateTime implements DateTimeInterface, so one would think that checking for DateTimeInterface is enough. Don't you think this deserves a comment? Or are you just suggesting that I only add it in one (which?) of the five classes?